src/ludamus/gates/web/django/urls.py

```
@@ -41,9 +41,7 @@ def healthz(
         with connection.cursor() as cursor:
             cursor.execute("SELECT 1")
         _healthz_cache.update(time=now, ok=True)
-    except (
-        Exception  # ruff:ignore[blind-except]  # pylint: disable=broad-exception-caught
-    ):
+    except Exception:  # ruff:ignore[blind-except]
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)